.github/workflows/main.yml

```diff
@@ -23,14 +23,12 @@ jobs:
       PYLINT_ARGS: '--output-format=parseable --errors-only'
       PYLINT_CHECK: true
       ROS_REPO: main
-      SECRET_AVAILABLE: ${{secrets.GITHUBPAT}}
-      UNDERLAY: '/root/git/care-o-bot/devel'  # workspace on docker image
 
     strategy:
       matrix:
         include:
-          - { ROS_DISTRO: kinetic, DOCKER_IMAGE: 'ghcr.io/mojin-robotics/mojin-ros:kinetic-full' }
-          - { ROS_DISTRO: noetic, DOCKER_IMAGE: 'ghcr.io/mojin-robotics/mojin-ros:noetic-full' }
+          - { ROS_DISTRO: kinetic, UPSTREAM_WORKSPACE: .ci.rosinstall.kinetic }
```

Comment:
@benmaidel apparently, the people of `roscontrol_boilerplate` made some updates to `kinetic-devel` - probably in preparation to release as requested in https://github.com/PickNikRobotics/ros_control_boilerplate/issues/30
see commits https://github.com/PickNikRobotics/ros_control_boilerplate/commits/kinetic-devel
however, now `kinetic-devel` does not seem to be in sync with `noetic-devel` anymore....could you have a look?

Reply:
the solution to changes in upstream repos is to use mojin-robotics forks. So we have control over what and when to update!

Reply:
apparently, we have been using `mojin-robotics/ros_control_boilerplate#kinetic-devel` in all other repos' rosinstall file - except for `precise_ros` itself

the change in the API is minimal: https://github.com/mojin-robotics/precise_ros/pull/20/commits/6260f2e1251f8f6e3f1243737fa1779b13a7f619

let's see what the answer to https://github.com/PickNikRobotics/ros_control_boilerplate/issues/30#issuecomment-782690317 will be

if they release soon, I'd rather get rid of the overlay completely